app/routes/user.py

```diff
@@ -0,0 +1,41 @@
+from flask import request , jsonify , Blueprint ,current_app
+import jwt
+from app.model import User,Role
+from app import db
+
+def token_access(f):
+      def decorator(*args, **kwargs):
+            payload=request.headers["Authorization"]
+            try:
+                  payload = payload.split(" ")[1]
+                  decoded_jwt=jwt.decode(payload, current_app.config.get('SECRET_KEY'), algorithms=["HS256"])
+                  user_id=decoded_jwt['user_id']
+                  user_role_id=decoded_jwt['user_role_id']
+                  print("heloos")
+                  
+
+            except  Exception as err:
+                  print("your error is ",err)
+                  return jsonify({"message": "Invalid token!"})
+            return f(user_id,user_role_id, *args, **kwargs)
+      return decorator     
+               
+
+
+
+user_profiles=Blueprint("/userprofile",__name__)
```

Comment:
The route should be like
/user/profile
/user/list
/user/create

So, Choose Blueprint prefix and endpoint names correctly